rfmsrc/common/rfm.h

```diff
@@ -391,10 +391,10 @@
 #define RFM_TX_CONTROL_MOD(baud)	(((baud)<8000) ? \
 									RFM_TX_CONTROL_MOD_45 : \
 									( \
-										((baud)<20000) ? \
+										((baud)<30000) ? \
 										RFM_TX_CONTROL_MOD_60 : \
 										( \
-											((baud)<30000) ? \
+											((baud)<40000) ? \
 											RFM_TX_CONTROL_MOD_75 : \
 											( \
 												((baud)<40000) ? \

```

Comment:
What about this value? Even with the later commit fixing this part, there seems to be two sections for 40000, thus one is useless. Probably not correct.
BTW, is there any "science" behind changing these numbers, or just trial and error tuning?

Reply:
Yes you are right, I forgot to pick it from my local branch.